controllers/agents.js

```diff
@@ -957,7 +957,7 @@ router.delete('/', function(req, res) {
         return;
     }
 
-    data_request['arguments']['purge'] = 'purge' in req.body && (req.body['purge'] == true || req.body['purge'] == 'true');
+    data_request['arguments']['purge'] = 'purge' in req.query && (req.query.purge == 'true' || req.query.purge == '');
```

Comment:
If the `purge` argument is present in the **query**:
* It can be `'true'`.
* It can be empty.

If it is present in the **body**:
* It can be `true`.
* It can be `'true'`.

Reply:
Let's allow the purge in both places: query and body. The most restrictive wins.